tracker-card.js

```diff
@@ -128,6 +134,13 @@ class TrackerCard extends HTMLElement {
           });
           this.handlers['custom_updater'] = true;
         }
+        if(window.location.href.indexOf('#install') > 0){
```

Comment:
This looks like a hack. What are you trying to achieve? Using href like this and doing an alert is far from ideal. 
I would make a parser function at the very least, and use some other way to signal installation complete.

Reply:
The reason it looks like a hack is probably because it is? :)

But this is the only way I could figure out how to include a install line per. card/component, that could pass the name to the install function in the component 🤷‍♀️

Reply:
I'll have a look tomorrow and get a better alternative. Might still use `window.location.hash` but at least make a nicer parse and proper confirmation message with some fade logic.